frontend/src/serviceWorkerDev.js

```diff
@@ -0,0 +1,21 @@
+export default function registerServiceWorker() {
+  if ('serviceWorker' in navigator) {
+    const url = `${process.env.PUBLIC_URL}/serviceWorker.js`;
+    navigator.serviceWorker
+      .register(url, {
+        scope: '/webform',
+      })
+      .then((registration) => {
+        if (registration.installing) {
+          console.info('Service worker installing');
+        } else if (registration.waiting) {
+          console.info('Service worker installed');
```

Comment:
How we uninstall the service worker / do we have any trigger to reinstall the service worker (e.g check the website version) ?

Reply:
For now, we don't have a trigger for that. I will check for this and update this when I work on issue #274 .